### DISCUSSION TITLE
Hosting Requirements? Do I need a VPS to run this?

### DISCUSSION THREAD

**Discussion by bukit-kronik** [DustinBrett/daedalOS#335] [Q&A]
Very cool project but need specs for the hosting to run smoothly..

**Comment by DustinBrett** [comment]
This compiles down to JS/HTML/CSS only, so it can be hosted on anything that supports running a web server. The actual power to run anything happens client side once the code is downloaded and ran in the browser. I have ran my site client side on my Samsung S8 without issue. Server side I host it on StableHost, uploaded via FTP and CDN from CloudFlare.
